scrapers/src/extractors/office.rs

```diff
@@ -0,0 +1,321 @@
+                    title: "Board of Regents".into(), // TODO - Is this right?
+                    r#type: None,
+                    chamber: None,
+                    district_type: Some(db::DistrictType::School),
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::District,
+                },
+            ),
+            (
+                r"(?i:District Attorney)",
+                OfficeMeta {
+                    name: "District Attorney".into(),
+                    title: "District Attorney".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: Some(db::DistrictType::Judicial),
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::District,
+                },
+            ),
+            (
+                r"(?i:District Court Judge)",
+                OfficeMeta {
+                    name: "District Court Judge".into(),
+                    title: "District Court Judge".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: Some(db::DistrictType::Judicial),
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::District,
+                },
+            ),
+            (
+                r"(?i:County Court Judge)",
+                OfficeMeta {
+                    name: "County Court Judge".into(),
+                    title: "County Court Judge".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: None,
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::County,
+                },
+            ),
+            (
+                r"(?i:Court of Appeals Judge)",
+                OfficeMeta {
+                    name: "Court of Appeals Judge".into(),
+                    title: "Court of Appeals Judge".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: None,
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::State,
+                },
+            ),
+            (
+                r"(?i:Supreme Court Justice)",
+                OfficeMeta {
+                    name: "Supreme Court Justice".into(),
+                    title: "Supreme Court Justice".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: None,
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::State,
+                },
+            ),
+            (
+                // TODO - should we have state only extractors? (Colorado RTD)
```

Comment:
can you explain what you mean?

Reply:
Yeah, the scope of this function is intended to be used by all the scrapers we implement. As it grows, I imagine we'll see a lot of offices that are specific to a municipality. Like Colorado's RTD, I don't think we'll see any other state have an office for "Regional Transportation District Director". So this comment was me kind of thinking out load about possibly having different tiers of extractors distinguished by region. I guess it's not particularly important to think about now.